src/components/sky-nav/sky-nav.twig

```diff
@@ -26,18 +22,25 @@
     <h2 id="menu-label" class="u-hidden-visually">
       Main Menu
     </h2>
-    {#
-      Preemptively adding `role="list"` since we will be removing list styles
-      via CSS
-      https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html
-    #}
+    {# Preemptively adding `role="list"` since we will be removing list styles
+    via CSS https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html #}
     <ul role="list">
       {% for item in menu.items %}
-        <li>
-          <a href="{{item.link}}"{% if item.current %} aria-current="page"{% endif %}>
-            {{item.title}}
-          </a>
-        </li>
+      <li>
+        <a
+          href="{{ item.link }}"
+          {%
+          if
+          item.current
+          %}
+          aria-current="page"
+          {%
+          endif
+          %}
+        >
+          {{ item.title }}
+        </a>
+      </li>
```

Comment:
I don't understand these formatting changes.

If this happened automatically, I think you should really consider simplifying your code editor extensions and/or settings. Something is funky.

If you don't know why it's happening, I would steal some time from someone in our dev team to screen share and help you troubleshoot.

Please take care not to request another review until formatting issues have been resolved.

Reply:
The formatting issues were being auto-generated by the twig extension I had installed. I am now using the same twig extension you share [here](https://gist.github.com/tylersticka/15d6944b449134f9077a962f4d1bdfad#gistcomment-3275614) and I have reviewed the Files Changed tab and believe this is ready for review again. 

I do apologize about prior review cycles.